pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+name = "streamlit-dataframe-summary"
+version = "0.1.0"
+description = "Streamlit component to summarize a DataFrame"
+readme = "README.md"
+requires-python = ">=3.12"
```

Comment:
I think we could have less strict python requirement

Reply:
Yup, I'll soften that